tests/portstat_test.py

```diff
@@ -618,6 +627,187 @@ def teardown_class(cls):
+        )
+        logger.debug("result:\n{}".format(result.output))
+        logger.debug("return_code:\n{}".format(result.exit_code))
+
+        assert result.output == output[format]
+        assert result.exit_code == SUCCESS
+
+        cmd = ['portstat', '--trim', '-i', intf]
+
+        if format == "json":
+            cmd.append('-j')
+
+        return_code, result = get_result_and_return_code(cmd)
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        assert result == output[format]
+        assert return_code == SUCCESS
+
+    def test_show_port_trim_counters_period(self):
+        runner = CliRunner()
+
+        result = runner.invoke(
+            show.cli.commands["interfaces"].commands["counters"].commands["trim"],
+            ["-p", str(TEST_PERIOD)]
+        )
+        logger.debug("result:\n{}".format(result.output))
+        logger.debug("return_code:\n{}".format(result.exit_code))
+
+        assert result.output == assert_show_output.trim_counters_period
+        assert result.exit_code == SUCCESS
+
+        return_code, result = get_result_and_return_code(
+            ['portstat', '--trim', '-p', str(TEST_PERIOD)]
+        )
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        assert result == assert_show_output.trim_counters_period
+        assert return_code == SUCCESS
+
+    def test_clear_port_trim_counters(self):
+        # Clear counters
+        return_code, result = get_result_and_return_code(
+            ['portstat', '-c']
+        )
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        assert result == assert_show_output.trim_counters_clear_msg
+        assert return_code == SUCCESS
+
+        # Verify updated stats
+        return_code, result = get_result_and_return_code(
+            ['portstat', '--trim']
+        )
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        verify_after_clear(result, assert_show_output.trim_counters_clear_stat.rstrip())
+        assert return_code == SUCCESS
+
+        # Verify raw stats
+        return_code, result = get_result_and_return_code(
+            ['portstat', '--trim', '--raw']
+        )
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        assert result == assert_show_output.trim_counters_all
+        assert return_code == SUCCESS
+
+        # Verify stats after snapshot cleanup
+        return_code, result = get_result_and_return_code(
+            ['portstat', '--trim', '-d']
+        )
+        logger.debug("result:\n{}".format(result))
+        logger.debug("return_code:\n{}".format(return_code))
+
+        assert result == assert_show_output.trim_counters_all
+        assert return_code == SUCCESS
```

Comment:
Please add tests for portstat and queuestat clear

Reply:
@kperumalbfn please have a look at `test_clear_port_trim_counters` and `test_clear_queue_trim_counters`